AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -225,34 +231,76 @@ - (NSError *)generateDisabledError:(NSString *)operation documentId:(NSString *)
 - (void)readWithPartition:(NSString *)partition
                documentId:(NSString *)documentId
              documentType:(Class)documentType
-              readOptions:(MSReadOptions *_Nullable)__unused readOptions
+              readOptions:(MSReadOptions *_Nullable)readOptions
         completionHandler:(MSDocumentWrapperCompletionHandler)completionHandler {
   @synchronized(self) {
     if (![self canBeUsed] || ![self isEnabled]) {
       NSError *error = [self generateDisabledError:@"read" documentId:documentId];
       completionHandler([[MSDocumentWrapper alloc] initWithError:error documentId:documentId]);
       return;
     }
-    [self performOperationForPartition:partition
-                            documentId:documentId
-                            httpMethod:kMSHttpMethodGet
-                                  body:nil
-                     additionalHeaders:nil
-                     completionHandler:^(NSData *_Nullable data, NSHTTPURLResponse *_Nullable __unused response,
-                                         NSError *_Nullable cosmosDbError) {
-                       // If not created.
-                       if (!data || [MSDataSourceError errorCodeFromError:cosmosDbError] != MSACDocumentSucceededErrorCode) {
-                         MSLogError([MSDataStore logTag], @"Not able to read the document ID:%@ with error:%@", documentId,
-                                    [cosmosDbError localizedDescription]);
-                         completionHandler([[MSDocumentWrapper alloc] initWithError:cosmosDbError documentId:documentId]);
-                         return;
-                       }
+  }
+  // FIXME: Check real network connection instad of offline mode.
```

Comment:
This is not exactly the logic we have in mind: https://msmobilecenter.visualstudio.com/Mobile-Center/_wiki/wikis/Mobile-Center.wiki?wikiVersion=GBwikiMaster&pagePath=%2FHome%2FTeams%2FMBaaS%2FStorage%2FSpecifications%2FLocal%20Documents%20Storage&pageId=2128&anchor=generic-operation-implementation.